xbmc/cores/DataCacheCore.cpp

```diff
@@ -26,7 +26,8 @@ CDataCacheCore::CDataCacheCore() :
   m_playerVideoInfo {},
   m_playerAudioInfo {},
   m_renderInfo {},
-  m_stateInfo {}
+  m_stateInfo {},
```

Comment:
can we do this in the header? imo much better than having ever growing ctors.

Reply:
i prefer it. i have tried that earlier and been told no init in in header.

Reply:
done.

Reply:
thanks